Assets/Prefabs/UI/TempUI.prefab

```diff
@@ -0,0 +1,272 @@
+%YAML 1.1
```

Comment:
What does a TempUI do?

Reply:
Just a textbox with the current temporary controls. This will be replaced later on.